src/services/api.service.js

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable camelcase */
+import axios from 'axios';
+import { ACCESS_TOKEN_STORAGE_KEY } from '../config/token.config';
+
+const API_BASE_URL = 'http://localhost:4000/api/v1';
+
+// const API_BASE_URL = 'http://localhost:3000/api/v1';
```

Comment:
- [ ] Your app needs the server to be running with a special port (4000) instead of the classic (3000).  Kindly add this to your readme.
- [ ] Kindly remove the commented code.

## Tips and extra help

The correct way to run the rails server on port 4000 is:

```
rails server -p 4000
```

That way, your users will know the correct port to run the server, so the API can be consumed correctly by the frontend.  💪